packages/components/src/aria-types.ts

```diff
@@ -187,4 +187,75 @@ export type AriaAttributes = {
   'aria-valuenow'?: number | undefined;
   /** Defines the human readable text alternative of aria-valuenow for a range widget. */
   'aria-valuetext'?: string | undefined;
-};
\ No newline at end of file
+  role?: AriaRole; };
```

Comment:
Could add a new line before `};` for better formatting :)
Also should we add a comment before? Is this visible anywhere in the IDE?

Reply:
Yes, for sure! Done :-)